hack/maybe-dci-check-change

```diff
@@ -36,10 +36,17 @@ proj=$(sed -e 's@https://github.com/\(.*\)/pull/.*@\1@i' <<< $URL)
 json=$(curl -s -L "${HEADERS[@]}" https://api.github.com/repos/$proj/pulls/$pr)
 statuses_url=$(jq -r .statuses_url <<< $json)
 
-# Check if there is a code change
-if curl -s -L "${HEADERS[@]}" "https://api.github.com/repos/$proj/pulls/$pr/files"|jq -r .[].filename|grep -v '\.md$'|grep -E 'roles/|plugins/'; then
+# check if there is a Test-Hints: force-check string in the PR description
+if grep -iq '^\s*Test-Hints:\s*force-check' <<< $(jq -r .body <<< $json); then
+    force_check=1
+else
+    force_check=
+fi
+
+# Check if there is a force-check or a code change
+if [ -n "$force_check" ] || curl -s -L "${HEADERS[@]}" "https://api.github.com/repos/$proj/pulls/$pr/files"|jq -r .[].filename|grep -v '\.md$'|grep -E 'roles/|plugins/'; then
     curl -s -L "${HEADERS[@]}" -X POST -d "{\"state\":\"pending\",\"context\":\"$GITHUB_JOBNAME\",\"description\": \"QUEUED\"}" "$statuses_url"
-    dci-check-change --silent "$URL"
+    dci-check-change "$URL"
 else
     curl -s -L "${HEADERS[@]}" -X POST -d "{\"state\":\"success\",\"context\":\"$GITHUB_JOBNAME\",\"description\": \"non code files\"}" "$statuses_url"
```

Comment:
Can we use the same description as in the PR? `"No code change"`

Reply:
right and also to be the same as in dci-merge.sh.